### DISCUSSION TITLE
Add login/mdp credentials to log in to existing Supabase dashboard using Github SSO

### DISCUSSION THREAD

**Discussion by Blockost** [supabase/supabase#21722] [Questions] [answered]
Hi,

I currently log in to my Supabase account on supabase.com via Github SSO.

Is it possible to add a password to the account to be able to log in via email/pwd in addition to Github SSO?

To reiterate, I'm not trying to add multiple login schemes to an app that relies on  Supabase Auth. It's for my admin Supabase account to manage the Supabase dashboard.

Thanks for your help and for this amazing product 🔥

**Comment by roine** [comment]
You can just call `await supabase.auth.updateUser({password: userPWD})`, this will add a password to the user and he/she'll be able to login using the password. I've got it working on my app.

**Reply by Blockost** [reply]
Hi @roine, thanks for your help on this but  as I said, I'm not trying to do it within an app relying on Supabase Auth. It's for my own Supabase admin account.

But you message reminded that this question is still open whereas it has been solved a few months ago by contacting Supabase support directly. 

I'll close the ticket.

**Comment by Blockost** [comment, marked as answer]
I contacted Supabase support directly and this was their answer (from March, 3rd 2024):

> There is no simple way to change authentication methods. One way is to create backups of your projects, delete your active projects then once done, we can delete your Supabase account that uses GitHub Auth before you'll be able to create it again with the new Auth flow.
>
>
> Once your new account using Email auth has been created. You can restore/import your projects using PSQL.
>
>
> We’re working to make user account deletion a self-serve process from the dashboard so, stay tuned.

So, deleting (via a support request) and recreating an account is the only way at the moment.
